pkg/notes/adaptor/repository/prisma.ts

```diff
@@ -0,0 +1,242 @@
+import { Option, Result } from '@mikuroxina/mini-fn';
+import type { PrismaClient } from '@prisma/client';
+
+import type { AccountID } from '../../../accounts/model/account.js';
+import type { ID } from '../../../id/type.js';
+import { Bookmark } from '../../model/bookmark.js';
+import { Note, type NoteID, type NoteVisibility } from '../../model/note.js';
+import type {
+  BookmarkRepository,
+  NoteRepository,
+} from '../../model/repository.js';
+
+interface deserializeNoteArgs {
```

Comment:
```suggestion
interface DeserializeNoteArgs {
```
PascalCase にしておくべきかと